sdk/tests/core/test_az_credential_client_secret.c

```diff
@@ -17,22 +17,34 @@
 
 #include <azure/core/_az_cfg.h>
 
+static az_span authority = AZ_SPAN_LITERAL_FROM_STR("https://login.microsoftonline.com/");
+
 static void test_credential_client_secret(void** state)
 {
   (void)state;
+  az_span const authorities[] = { AZ_SPAN_FROM_STR("https://login.microsoftonline.com/"),
+                                  AZ_SPAN_FROM_STR("https://somesite.contoso.com/") };
```

Comment:
Can we structure the tests so we can also validate that the user-provided authority is retained correctly. I don't see anywhere we are asserting that we still have `"https://somesite.contoso.com/"` in the `request_url`.

Reply:
We do -- see `assert_true(az_span_is_content_equal(az_auth_url, request_url));` - and this is why we should not be renaming `authority` to `msft_authority`.